optimizer_wrapper.rb

```diff
@@ -101,7 +101,7 @@ def self.wrapper_vrp(api_key, services, vrp, checksum, job_id = nil)
       raise UnsupportedProblemError.new('Cannot apply any of the solver services', inapplicable_services)
     elsif config[:solve][:synchronously] || (
             services_vrps.size == 1 &&
-            !vrp.preprocessing_cluster_threshold &&
+            vrp.preprocessing_cluster_threshold.to_f == 0.0 &&
```

Comment:
I think the previous version is correct, no? 

It has been some time but now that zip_clustering will be active when threshold is 0, we don't want to activate it for 0 and we want async only for `nil`

Reply:
I could not find the comment but I remember that we said that if cluster_threshold is zero then zip_cluster would be fast enough to be solve synchronously if other parameters let it.

Still, I agree it is easier for us to know that zip_clusters is always callend in async. Moreover, it is safer because we might add cost_full checks or operations in zip_clusters without noticing they will be done synchronously.

Reply:
Now that you mention I vaguely remember (we first had to wait for mt-web defaults to get corrected and then this PR fell in the cracks).

Yeah, I think your argument is even stronger so lets skip synch if it is zip_cluster